PythonVisualizations/Stack.py

```diff
@@ -0,0 +1,322 @@
+import time
+from tkinter import *
+
+try:
+    from drawable import *
+    from VisualizationApp import *
+except ModuleNotFoundError:
+    from .drawable import *
+    from .VisualizationApp import *
+
+class Stack(VisualizationApp):
+    nextColor = 0
+    MAX_ARG_WIDTH = 8
+    MAX_SIZE = 8
+    CELL_WIDTH = 20 * MAX_ARG_WIDTH
+    CELL_HEIGHT = 40
+    CELL_BORDER = 2
+    CELL_BORDER_COLOR = 'black'
+    STACK_X0 = 300
+    STACK_Y0 = 400
+
+    def __init__(self, size=MAX_SIZE, maxArgWidth=MAX_ARG_WIDTH, title="Stack", **kwargs):
+        super().__init__(title=title, maxArgWidth=maxArgWidth, **kwargs)
+        self.size = size
+        self.title = title
+        self.list = []
+        self.buttons = self.makeButtons()
+        self.maxArgWidth = maxArgWidth
```

Comment:
I think `self.maxArgWidth` needs to be set before calling `self.makeButtons()` which calls `self.addOperation()`.   In `self.addOperation()`, the text entry widgets are created, and that's where the argument width needs to known.